templates/letsencrypt.service.erb

```diff
@@ -0,0 +1,10 @@
+[Unit]
```

Comment:
please add a comment at the top like `# THIS FILE IS MANAGED BY PUPPET`

Reply:
Done.